content/post/gld.md

```diff
@@ -0,0 +1,1363 @@
+---
+author: vadim
+date: 2018-04-17
+title: "go-license-detector"
```

Comment:
I don't think this should be the title.
What about: "Detecting licenses in code with Go and ML"?